### DISCUSSION TITLE
10-04-2024 Steering Committee Meeting Minutes

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#128] [Steering Committee]
1. Adoption of Tally for DCDAO: Agreed to proceed with integrating Tally for DCDAO operations, will probably to replace NFT contract with a new one for integration and support vote. Request @casinoyoda and @hujw77 to follow up on this.

2. A new [proposal](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x2e17e91c48afaf8233e5b9fed3ea48cc3b572a7f36d895732619fd64d5fa5331) has been submitted for the next Steering Committee.

3. Review and approve https://github.com/dcdao/profiles/issues/58 Request @casinoyoda to mint the membership NFT

4. HydraDX Omnipool listing and integration of RING
 - https://darwinia2.subsquare.io/referenda/2
 - https://darwinia.subscan.io/extrinsic/2449352-0
 - Request Multisig Team to create a multisig on HydraDX for further usage @casinoyoda , and request @2xJDubs, the proposer of referenda 2 to draft the operation guide for the next steps.

5. Review [report](https://github.com/orgs/dcdao/discussions/126) and GTM plan from @PartnershipDude https://github.com/orgs/dcdao/discussions/126 , giving feedbacks.

**Comment by 2xJDubs** [comment]
@hackfisher I have been speaking with the HydraDX team and they will be initiating an HRMP accept and open request this week.  The ywere unaware of this referendum.  https://darwinia2.subsquare.io/referenda/0  Doing this will also add RING as an asset to HydraDX.  There is still teh on-going discussion around adding RING to the Omnipool but it has all been favorable so far.

As for the Operations guide for next steps, I will continue my discussion with the HydraDX team on what the steps are once RING is approved and provide them to the Multisig team.

**Comment by hackfisher** [comment]
DCDAO multisig(3/5) `7L4Wh6L7qWNQF2BBEjWDShhEafDLtEVzaki3rYTWSztmPsY2` on HydraDX:

N: `7J4ssTZfmT4NFPmXdeMTzksho2TJqyzQtYGdCcx2x17q6oFT`
Ec: `7J7hVPqgR5uHGHg3b75mEHHyU28VbR5Zv7dUxffjNNTXaRyZ`
Ev: `7KRBP67vyHGdjktGx9Nsy5LWFmaXcPoTr23RSnTEMskEoiuj`
R: `7L9XPTFNJLH7zEBGoLR44gHozV64NBkm6kPLUCJcfj5eVcaf`
A: `7M6cyUbiaGAb1VnotcPkgxfcepWi3mtvtM6P17tqQvqbeTzn`


Using https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fhydradx.paras.ibp.network#/accounts
